Essentials/src/com/earth2me/essentials/Kit.java

```diff
@@ -163,7 +163,7 @@ public void expandItems(final User user, final List<String> items) throws Except
                 if (kitItem.startsWith("/")) {
                     String command = kitItem.substring(1);
                     String name = user.getName();
-                    command = command.replace("{player}", name);
+                    command = command.replace("{player}", name).replace("{PLAYER}", name);
```

Comment:
```suggestion
                    command = command.replaceAll("{(?:player|PLAYER)}", name);
```

Bonus points if you make the regex a static variable to avoid repeated compilation of the regex.